src/main/java/org/folio/linked/data/model/entity/event/ResourceAuthorityCreatedEvent.java

```diff
@@ -0,0 +1,4 @@
+package org.folio.linked.data.model.entity.event;
+
+public record ResourceAuthorityCreatedEvent(Long id) {
```

Comment:
We could omit the 'Resource' word as for me, AuthorityCreatedEvent name contains all required info.
But also, I think we better remove this event class at all (see the comment in ResourceService)